spec/models/email_alert_signup_spec.rb

```diff
@@ -0,0 +1,80 @@
+require "spec_helper"
+
+describe EmailAlertSignup do
+
+  let (:content_item) {
+    OpenStruct.new(
+      title: "Employment",
+      description: "People like getting paid",
+      details: OpenStruct.new(
+        tags: OpenStruct.new(
+          policy: ["employment"]
+        ),
+        breadcrumbs: [
+          OpenStruct.new(title: "Economy", link: "https://www.gov.uk/government/policies/economy"),
+          OpenStruct.new(title: "The UK", link: "https://www.gov.uk"),
+        ]
+      )
+    )
+  }
+
+  let (:email_alert_api) { double(:email_alert_api) }
+
+  before do
+
+    EmailAlertFrontend.register_service(:email_alert_api, email_alert_api)
+
+    allow(email_alert_api).to receive(:find_or_create_subscriber_list).and_return(OpenStruct.new("subscriber_list" => OpenStruct.new("subscription_url" => "http://govdelivery_signup_url")))
+  end
+
+  it "is invalid with no content item" do
+    expect(EmailAlertSignup.new(nil)).not_to be_valid
+  end
+
+  describe "#save" do
+    it "creates the topic in GovDelivery using the tag and title" do
+      email_signup = EmailAlertSignup.new(content_item)
+
+      allow(email_alert_api).to receive(:find_or_create_subscriber_list).with(
```

Comment:
Again, this just sets a stub, but does not follow it up with an expectation that the thing was actually called.